azkaban-common/src/test/java/azkaban/utils/TestUtils.java

```diff
@@ -77,4 +80,10 @@ public static UserManager createTestXmlUserManager() {
     return manager;
   }
 
+  public static String read(final String file, final Object parent) throws Exception {
```

Comment:
readResouceFile to be more descriptive?

Reply:
It would be more descriptive.. But, I wanted to keep this short to not bloat too much the test code that use this.

IMHO "read" is clear enough when you look at the tests that are calling this. I'd tend to think that a more descriptive name would be worth the added verbosity if this would be more like a public interface (where you for example let your IDE's auto-complete show what methods there are to pick on this class).

But it's your call, obviously. Do you want it renamed?

Reply:
Thanks for sharing your thoughts. but it would be best to rename and letting intellij's refactor to do renaming is pretty much effortless.